test/models/concerns/hostext/search_test.rb

```diff
@@ -106,6 +106,42 @@ class SearchTest < ActiveSupport::TestCase
           assert_not_includes result, other_host
         end
       end
+
+      context "search by facts" do
+        let (:host) { FactoryBot.create(:host) }
+        let (:name) { FactoryBot.create(:fact_name, :compose => true) }
```

Comment:
Can you rename this please? `name` shadows an internal method, that's why tests fail.